these fail with mimalloc since they test alloc failure:
../../../../src/gl/tests/test-calloc-gnu.c:64: assertion 'errno == ENOMEM' failed
FAIL test-calloc-gnu (exit status: 134)
--
--- a/src/gl/tests/Makefile.am
+++ b/src/gl/tests/Makefile.am
@@ -191,8 +191,6 @@
 
 ## begin gnulib module calloc-gnu-tests
 
-TESTS += test-calloc-gnu
-check_PROGRAMS += test-calloc-gnu
 EXTRA_DIST += test-calloc-gnu.c macros.h
 
 ## end   gnulib module calloc-gnu-tests
@@ -1661,16 +1659,12 @@
 
 ## begin gnulib module realloc-gnu-tests
 
-TESTS += test-realloc-gnu
-check_PROGRAMS += test-realloc-gnu
 EXTRA_DIST += test-realloc-gnu.c macros.h
 
 ## end   gnulib module realloc-gnu-tests
 
 ## begin gnulib module reallocarray-tests
 
-TESTS += test-reallocarray
-check_PROGRAMS += test-reallocarray
 EXTRA_DIST += test-reallocarray.c signature.h macros.h
 
 ## end   gnulib module reallocarray-tests
